L1Trigger/TrackFindingTracklet/src/InputRouter.cc

```diff
@@ -74,6 +74,7 @@ void InputRouter::execute() {
         iadd++;
       }
     }
-    assert(iadd == 1);
+    if (!(settings_.reduced()))
```

Comment:
I think the comment at L69 is wrong. Surely the InputRouter writes to the InputLinkMemory, not the AllStubs memories? And why is L77 required. In your reduced chain, do some InputRouter write to no output memories? This would seem odd.

Reply:
@trholmes  any answer to this comment?